RJL.HW4.OOP.Classes/Program.cs

```diff
@@ -119,25 +118,33 @@ public CRTTV(string model, string power, string diagonal, string frequency)
         }
         public string GetDescription()
         {
-            string description = $"model {this.Model}, power {this.Power}, diagonal {this.Diagonal}, scanning frequency {this.Frequency}";
-            return description;
+            return $"model {this.Model}, power {this.Power}, diagonal {this.Diagonal}, scanning frequency {this.Frequency}";
         }
     }
     public class Player
     {
         public string Model { get; private set; }
         public string Power { get; private set; }
-        public string Formats { get; private set; }
-        public Player(string model, string power, string formats)
+        public Format Formats { get; private set; }
```

Comment:
when we call property as a plural (`Formats `) we should provide a collection (array or whatever you want) instead of one instance